src/recommendpolicy/recommendPolicy.go

```diff
@@ -144,7 +162,7 @@ func RecommendPolicyMain() {
 			DeployNsName = append(DeployNsName, *deploy)
 		}
 
-		for _, ns := range nsNotFilterSysPolicy {
+		for _, ns := range nsNotFilter {
 			if d.Namespace != ns {
 				generateHardenPolicy(d.Name, d.Namespace, d.Spec.Template.Labels)
```

Comment:
We are already generating policies for `Deployment`s here. Why are we generating them again at line 150? It should be in a single place only.